### PR TITLE
feat: add route to download source file

### LINKED ISSUE #9: Add route to download source file
Given a token provided via query parameter a GET request to that route should return the attachment data.

Similar to what we've [done in mnutrition](https://github.com/eHealthAfrica/mnutrition-importer/blob/develop/lib/routes/data-reports-handlers/source.js).

### PR DESCRIPTION
**Pull request by jo**
`/api/submissions/{id}/source`: id is the `raw-data` docid.

This downloads the first attachment found in the document.

Closes #9

### CODE REVIEW

**Comment by gr2m** [comment]
How about we do `GET /api/submissions/{id}/attachments/{filename}`?

**Comment by jo** [comment]
In VAN we always have a single attachment, thats why I unburdened the client from the work to find the right attachment. But it was just my poor understanding of Kazanas architecture. Now that you asked and I had a deeper look into how that raw data thing works I absolutely agree that the route you purpose is the way to go - the only way which makes sense ;)

**Comment by peritus** [comment]
> How about we do `GET /api/submissions/{id}/attachments/{filename}`?
:+1:

**Comment by jo** [comment]
done

**Comment by peritus** [comment]
LGTM

File: lib/routes/submissions/source.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  method: 'GET',
+  path: '/api/submissions/{id}/source/{filename}',
```
Comment (gr2m) [inline lib/routes/submissions/source.js]: I'd rename `/source/` to `/attachments/` to make it more generic. The attachment is most often the source of the raw data, but that might not always be the case. 

Thoughts @jo @peritus?

Comment (jo) [inline lib/routes/submissions/source.js]: :+1:

**Pull request merged**